src/circle.css.ts

```diff
@@ -2,14 +2,13 @@ import {style} from '@vanilla-extract/css';
 import {sprinkles} from './sprinkles.css';
 
 export const circle = style([
+    {
+        boxSizing: 'border-box',
+    },
```

Comment:
the border was adding 2px to the size, we don't want this. Hopefully this doesn't break many screenshots in webapp